src/Component/Locations.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios'
+class Locations extends React.Component {
+    constructor(){
+        super();
+        this.state = {
+            cities:[],
+            showCity: false
+        }
+    }
+
+    ghibliCities = async () =>{
+        try{
+       const { data } = await axios.get("https://ghibliapi.herokuapp.com/locations")
+    
+       this.setState({cities:data})
+        } catch (e){
+            console.log(e)
+        }
+    }
+
+    componentDidMount(){
+        this.ghibliCities()
+    }
+    handleClick = () =>{
+        if(this.state.showCity === true){
```

Comment:
You can also write this line as:
`if(this.state.showCity){`
You don't have to be explicit for conditional statements. It's a little cleaner this way